src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1,3 +1,8 @@
+using Random
```

Comment:
This is a "new requirement", but could you qualify `using` and limit to the specific `Random` functions? (doing this across the board is now the norm, except for `using LightGraphs`).